### ISSUE TITLE
Preview is broken as well as the editor for putting the pictures as the background. wont show up when downloaded

### ISSUE THREAD

**Issue by YukimaSinji** [Gibbu/BDThemeEditor#51]

**Comment by xxxkingxl** [comment]
true, I have seen it myself, for example: just pull up the clearvision dc theme from the list of tamplates & u see background is missing:

![Image](https://github.com/user-attachments/assets/d3c622df-4924-4273-9648-0dee2bc41c2d)
its for sure broken
&
I cant upload dc theme css files to the editor, sinds it just ignores my upload (these 2 things are tested in: 1. Opera GX, 2. Edge, & Google Chrome (I am sorry I needed to pull up google chrome (where of I deleated the acces to on my pc, sinds I hate Google Chrome )))

**Comment by xxxkingxl** [comment]
I would say: confirmed
